kgforge/specializations/models/rdf/service.py

```diff
@@ -128,18 +143,34 @@ def lookup_shape_from_node(self, node: URIRef) -> Shape:
 
 class RdfService:
 
-    def __init__(self, graph: Graph, context_iri: Optional[str] = None) -> None:
+    def __init__(self, graph: RDFDataset, context_iri: Optional[str] = None) -> None:
 
         if context_iri is None:
             raise ConfigurationError("RdfModel requires a context")
         self._graph = graph
```

Comment:
Wouldn't it be better to have a more evocative name for _graph? Like graph_dataset?

Reply:
Let go for _dataset_graph